infra/app/envs/dev/main.tf

```diff
@@ -52,13 +52,20 @@ module "project_config" {
 }
 
 module "app" {
-  source                = "../../env-template"
-  environment_name      = local.environment_name
+  source           = "../../env-template"
+  environment_name = local.environment_name
+
+  # Image tags
   participant_image_tag = var.participant_image_tag
   staff_image_tag       = var.staff_image_tag
   analytics_image_tag   = var.analytics_image_tag
-  analytics_enable_exec = true
-  participant_url       = "${local.environment_name}.wic-services.org"
-  staff_url             = "${local.environment_name}-staff.wic-services.org"
-  analytics_url         = "${local.environment_name}-analytics.wic-services.org"
+
+  # Urls
+  participant_url = "${local.environment_name}.wic-services.org"
+  staff_url       = "${local.environment_name}-staff.wic-services.org"
+  analytics_url   = "${local.environment_name}-analytics.wic-services.org"
+
+  # Misc settings
+  participant_doc_upload_expiration = "60"
```

Comment:
.. maybe a little longer than 60 seconds? It's gonna be freaky to see thumbnails randomly disappear in dev..

Reply:
Good call. I'll bump it up to 5m